vtex-apps/structured-data
# Product list structured data

**What problem is this solving?**

This PR implements a component for product list structured data, according to: https://developers.google.com/search/docs/data-types/carousel

**How should this be manually tested?**

You can use this component in a page and check using Googles Rich Results Test tool.
Results from a test workspace, using this and this

Category page:
https://search.google.com/test/rich-results?id=IlRGTRjh46QsNGbY69Gifw

Home: 
https://search.google.com/test/rich-results?id=gfEyPpTJIeAiDjfaZCfmPQ

**Screenshots or example usage:**
You use the component and passes the product list as an array in its props, for example:
```
  return (
    <Fragment>
      <ProductList products={searchQuery.products} />
      {...children }
    </Fragment>
  )
}

```
